compiler/p/codegen/ControlFlowEvaluator.cpp

```diff
@@ -4518,67 +4525,14 @@ TR::Register *OMR::Power::TreeEvaluator::ZEROCHKEvaluator(TR::Node *node, TR::Co
 
    // Inline instructions for the check
    //
-   TR::Node *valueToCheck = node->getFirstChild();
-
-   if (valueToCheck->getOpCode().isBooleanCompare() &&
-       valueToCheck->getChild(0)->getOpCode().isIntegerOrAddress() &&
-       valueToCheck->getChild(1)->getOpCode().isIntegerOrAddress() &&
-       performTransformation(cg->comp(), "O^O CODEGEN Optimizing ZEROCHK+%s %s\n", valueToCheck->getOpCode().getName(), valueToCheck->getName(cg->getDebug())))
-      {
-      if (valueToCheck->getOpCode().isCompareForOrder())
-         {
-         if (valueToCheck->getChild(0)->getOpCode().is8Byte() ||
-             valueToCheck->getChild(1)->getOpCode().is8Byte())
-            {
-            // switch branches since we want to go to OOL on node evaluating to 0
-            // which corresponds to an ifcmp fall-through
-            TR::InstOpCode::Mnemonic branchOp = cmp2branch(valueToCheck->getOpCode().getOpCodeForReverseBranch(), cg);
-            TR::InstOpCode::Mnemonic reverseBranchOp = cmp2branch(valueToCheck->getOpCodeValue(), cg);
-            TR::TreeEvaluator::compareLongsForOrder(branchOp, reverseBranchOp, slowPathLabel, valueToCheck, cg,
-                                 valueToCheck->getOpCode().isUnsignedCompare(), true, PPCOpProp_BranchUnlikely);
-            }
-         else
-            {
-            // switch branches since we want to go to OOL on node evaluating to 0
-            // which corresponds to an ifcmp fall-through
-            TR::TreeEvaluator::compareIntsForOrder(cmp2branch(valueToCheck->getOpCode().getOpCodeForReverseBranch(), cg),
-                                slowPathLabel, valueToCheck, cg, valueToCheck->getOpCode().isUnsignedCompare(),
-                                true, PPCOpProp_BranchUnlikely);
```

Comment:
Did we lose the PPCOpProp_BranchUnlikely hint?

Reply:
I think this was resolved without any comment...

Reply:
My previous top-level comment addresses this, as this comment was on the `ZEROCHK` code that wasn't supposed to be included here. This code is no longer present in this PR, so this is no longer a problem. Sorry, I should have made this more clear.